### PR TITLE
How do I pronounce DNSimple

### PR DESCRIPTION
**Pull request by onlyhavecans**
We get this asked enough I thought it would be fun to have a support article about it.

### CODE REVIEW

**Comment by OleMchls** [comment]
I always explain that

> Here at DNSimple we love sharing, that's why DNS and Simple share the "S"

**Comment by onlyhavecans** [comment]
> It'd be nice to have a closing paragraph.

I'm dry for ideas on what kind of closing paragraph we would have on a support article. Can someone suggest something?

**Comment by OleMchls** [comment]
Here is my free idea for you:

> If your still unsure how to pronounce it, record yourself saying it out loud and give your award-winning support a try by sending along your recording and we will also help you out with this (and any other Domain Management automation question).

**Comment by onlyhavecans** [comment]
I added @OleMchls idea @aeden. Does this wrap up the support article better?

File: content/articles/how-is-dnsimple-pronounced.markdown [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+But pronoucing it isn't always simple.
+
+Here's a basic guide:
+
+## Pronouncing DNSimple
+
+The easiest way to visualize it is D-N-Simple.
+
+Here at DNSimple we love sharing, that's why DNS and Simple share the "S".
```
Comment (itsalyse) [inline content/articles/how-is-dnsimple-pronounced.markdown]: >Here at DNSimple we love sharing, that's why DNS and Simple share the "S"

This seems like something that makes more sense internally than externally. I think the initial explanation is enough without it.

**Pull request merged**